tests/fv3_conf/cpld_control_run.IN

```diff
@@ -105,6 +109,12 @@ else
       cp ../${DEP_RUN}${SUFFIX}/RESTART/${RESTART_FILE_PREFIX}.MOM.res_3.nc ./INPUT/MOM.res_3.nc
      else
       cp ../${DEP_RUN}${SUFFIX}/RESTART/${RESTART_FILE_PREFIX}.MOM.res.nc ./INPUT/MOM.res.nc
+#      if [[ ${IAU_OFFSET} == 6 ]] && [[ $FHROT <= 6 ]] ; then
+       if [[ ${IAU_OFFSET} == 6 ]] ; then
+       if [[ $FHROT -le 6 ]] ; then
+        cp /scratch1/NCEPDEV/nems/Jun.Wang/nems/vlab/inc/c96/ocn/mom6_increment.nc ./INPUT/mom6_increment.nc
```

Comment:
@jkbk2004 I need code managers to copy this data /scratch1/NCEPDEV/nems/Jun.Wang/nems/vlab/inc/c96/ocn/mom6_increment.nc  to: input-data-20221101/MOM6_IC